### PR TITLE
Translate 2.7.0-preview1 news post (zh_cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

File: zh_cn/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+* 新增了 `Enumerable#tally`，它会计算每个元素出现的次数。
+
+      ["a", "b", "c", "b"].tally
+      #=> {"a"=>1, "b"=>2, "c"=>1}
+
+## 性能改进
+
+* JIT [实验性质]
+
+  * 当优化假设不成功时，JIT 后的代码可能会被重新编译到优化程度较低的代码。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md]: extra leading spaces?

File: zh_cn/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+## 其他自 2.6 版本以來显著的变化
+
+* 现在进行块方法调用时，如果 `Proc.new` 和 `proc` 在没有 block 会产生警告。
+
+* `lambda` 在方法调用时如果没有 block 会产生区块错误。
+
+* Unicode 和 Emoji 版本从 11.0.0 更新至 12.0.0。[[Feature #15321]](https://bugs.ruby-lang.org/issues/15321)
+
+* 更新 Unicode 至 12.1.0 版本，新增对于新年好「令和」 U+32FF 的支持。[[Feature #15195]](https://bugs.ruby-lang.org/issues/15195)
```
Comment (bugtender) [inline zh_cn/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md]: `年好` should be `年号`

Comment (dsh0416) [reply zh_cn/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md]: Thank you for your reviewing. The problems mentioned have fixed.

File: zh_cn/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md (hunk 1 above)
Comment (dsh0416) [reply zh_cn/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md]: I'm sorry that I don't get it. Which space is the extra leading space?

Comment (JuanitoFatas) [reply zh_cn/news/_posts/2019-05-30-ruby-2-7-0-preview1-released.md]: Sorry my mistake, looks good to me.

**Comment by bugtender** [comment]
@dsh0416 would you mind fixing the commit message? (like PR title)

**Comment by JuanitoFatas** [comment]
Merged as https://github.com/ruby/www.ruby-lang.org/commit/c8acb231e117126e6f6abfd4a103cdd8a0b9bce3, commit message fixed.

**Pull request merged**